src/main/java/edu/project2/renders/Render.java

```diff
@@ -0,0 +1,11 @@
+package edu.project2.renders;
+
+import edu.project2.maze.Coordinate;
+import edu.project2.maze.Maze;
+import java.util.List;
+
+public interface Render {
```

Comment:
```suggestion
public interface Renderer {
```
В названиях классов лучше использовать существительные